src/module-elasticsuite-core/Model/Healthcheck/PackageVersionsMismatchCheck.php

```diff
@@ -56,7 +57,7 @@ public function __construct(
         array $packagesToCheck = [],
         int $sortOrder = 50
     ) {
-        parent::__construct($urlBuilder, $sortOrder);
+        parent::__construct($urlBuilder, $sortOrder, MessageInterface::SEVERITY_CRITICAL);
```

Comment:
Please add the severity is as an exposed constructor parameter, after the sortOrder.